oauth/provider/ldap/ldap_test.go

```diff
@@ -0,0 +1,36 @@
+package ldap
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+
+var mock = ldapconf{
+	id:  "123",
+	host: "ldap://ldap.forumsys.com",
```

Comment:
I would appreciate it if you would use [dockertest](https://github.com/ory-am/dockertest) to spin up an LDAP server and test against that one. Using a public LDAP is not reliable because the server could shut down or configuration could change.